index.html

```diff
@@ -12,33 +12,62 @@
     <link rel="stylesheet" href="assets/css/main.css" type="text/css"/>
     <link rel="stylesheet" href="assets/css/heading.css" type="text/css"/>
     <link rel="stylesheet" href="assets/css/reset.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/pets-love.css" type="text/css"/>
 </head>
 <body>
- <div class="container">
-  <header>
-   <nav>
-    <div class="hamburger-wrapper">
-     <div class="hamburger">
-       <div class="hamburger-box">
-         <div class="hamburger-inner"></div>
+  <div class="container">
+    <header>
+      <nav>
+        <div class="hamburger-wrapper">
+          <div class="hamburger">
+            <div class="hamburger-box">
+              <div class="hamburger-inner"></div>
+            </div>
+          </div>
         </div>
-      </div>
-     </div>
-    </nav>
-    <div class="header-photo"></div>
-    <img class="header-logo"src="assets/images/logo.png"/>
+      </nav>
+      <div class="header-photo"></div>
+      <img class="header-logo"src="assets/images/logo.png"/>
       <div class="header-social-media-wrapper">
-       <div class="header-social-media-icon">
-        <i class="fab fa-facebook-f"></i>
-       </div>
-       <div class="header-social-media-icon">
-        <i class="fab fa-twitter"></i>
-       </div>
-       <div class="header-social-media-icon">
-        <i class="fab fa-instagram"></i>
-       </div>
+        <div class="header-social-media-icon">
+          <i class="fab fa-facebook-f"></i>
+        </div>
+        <div class="header-social-media-icon">
+          <i class="fab fa-twitter"></i>
+        </div>
+        <div class="header-social-media-icon">
+          <i class="fab fa-instagram"></i>
+        </div>
       </div>
+    </div>
   </header>
+  <div class="inner-wrapper">
+  <section class="pets-love-wrapper">
+    <div class="pets-love-heading">
+      <h4 class="main-small-heading">Pet's Love</h4>
+     <h1 class="largest-heading">Russian<div>Blue Cat</div></h1>
+    </div>
+    <div class="pets-love-photo">
+     <img src="assets/images/the-cat-british.jpeg"/>
+    </div>
+    <div class="pets-love-description">
+      <div class="main-small-description">
+      <p>
+        <div>The first step to making your cat famous is</div>
+        <div>by building a steady fan base on social</div>
```

Comment:
Żadnych` <div>` w `<p>`, `<h1>` itd!
do tego używamy` <span>,` lub tworzymy nowy element `<p> `- zależnie od tego co chemy uzyskać. W tym przypadku chcesz użyć span.
Jak Cię interesuje czemu, to napisz później z pytaniem o SEO (dopiero jak skończysz review)